proxy.go

```diff
@@ -87,6 +88,11 @@ func (p *Proxy) notAllowedResponse(w http.ResponseWriter, _ *http.Request) {
 	w.WriteHeader(403)
 }
 
+func (p *Proxy) notAcceptableResponse(w http.ResponseWriter, _ *http.Request) {
+	log.Debug("not acceptable response")
+	w.WriteHeader(406)
```

Comment:
using the http constant makes it more readable! :) 
```suggestion
	w.WriteHeader(http.StatusNotAcceptable)
```

Reply:
Good point! Done.